services/ingest/tests/test_chat_engine.py

```
@@ -112,9 +115,11 @@ async def test_run_chat_turn_includes_tool_call_results():
         content=[MagicMock(text='{"deployments": []}')],
         is_error=False,
     )
+    received_org_ids = []
 
     @asynccontextmanager
-    async def fake_mcp_session():
+    async def fake_mcp_session(org_id):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `fake_mcp_session`

(ANN202)